web-animations-2/Overview.bs

```diff
@@ -969,6 +969,41 @@ Add the following sentence:
 
 </div>
 
+### Calculating the progress of an animation ### {#the-progress-of-an-animation}
+
+  An animation's <dfn export for=animation>progress</dfn> is the ratio of its
+   [=animation/current time=] to its [=associated effect end=].
+
+  <div algorithm="calculate animation progress">
+    The [=animation/progress=] of an [=animation=], |animation|, is calculated as follows:
+
+    <dl class=switch>
+
+    : If <em>any</em> of the following are true:
+      * |animation| does not have an [=animation/associated effect=], or
+      * |animation|'s [=animation/current time=] is an [=unresolved=] time value,
+    ::
+        |animation|'s [=animation/progress=] is null.
+    : If |animation|'s [=associated effect end=] is zero,
+    ::
+        : If |animation|'s [=animation/current time=] is negative,
+        :: |animation|'s [=animation/progress=] is zero.
+
+        : Otherwise,
+        :: |animation|'s [=animation/progress=] is one.
+    : If |animation|'s [=associated effect end=] is infinite,
+    ::
+        |animation|'s [=animation/progress=] is zero.
+    : Otherwise,
+    ::
+      <blockquote>
+        <code><a for="animation">progress</a> = min(max([=animation/current time=] / |animation|'s [=associated effect end=], 0), 1)
```

Comment:
I think the 0 and 1 are backwards here. It looks like the 1 is passed to min and the 0 to max, meaning that the progress will always be min(progress, 1) or just 1.